vra/structure.go

```diff
@@ -54,6 +54,24 @@ func compareUnique(s []interface{}) bool {
 	return j == len(s)
 }
 
+////diff returns the elements in 's1' that are not in 's2'
```

Comment:
Should this be removed?

Reply:
Yes. I was exploring the fix with diffSuppressFunc and customizedDiff, for which I added this but ended up not using finally.